manifests/b/BlenderFoundation/Blender/3.6.2/BlenderFoundation.Blender.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: BlenderFoundation.Blender
+PackageVersion: 3.6.2
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- 3ds
+- fbx
+- glb
+- gltf
+- obj
+- ply
+- stl
+Installers:
+- Architecture: x64
+  InstallerUrl: https://mirror.freedif.org/blender/release/Blender3.6/blender-3.6.2-windows-x64.msi

```

Comment:
```suggestion
  InstallerUrl: https://mirrors.ocf.berkeley.edu/blender/release/Blender3.6/blender-3.6.2-windows-x64.msi
```
Hi @unwillinglyawesome,

This Berkeley.edu link is the InstallerUrl that I get when starting at the PackageUrl (https://www.blender.org) and navigating through the Download links. Where did you find the freedif.org link?

Reply:
On blender.org, it chooses a random mirror is my guess.

Reply:
![image](https://github.com/microsoft/winget-pkgs/assets/129231322/90e3c087-ac49-4e2b-a8fe-17b7918e5544)